core/src/main/java/io/github/thibaultbee/streampack/internal/gl/EglWindowSurface.kt

```diff
@@ -35,77 +32,24 @@ import java.util.Objects
  * (Contains mostly code borrowed from CameraX)
  */
 
-class EglWindowSurface(private val surface: Surface, useHighBitDepth: Boolean = false) {
-    private var eglDisplay: EGLDisplay = EGL14.EGL_NO_DISPLAY
-    private var eglContext: EGLContext = EGL14.EGL_NO_CONTEXT
+class EglWindowSurface(
```

Comment:
I broke EglWindowSurface and EgDisplayContext into two different classes bc I need to create the texture outside the scope of any particular surface context.